src/Settings/VideoSettings.cc

```diff
@@ -29,6 +29,10 @@ const char* VideoSettings::videoSourceMPEGTS            = "MPEG-TS (h.264) Video
 const char* VideoSettings::videoSource3DRSolo           = "3DR Solo (requires restart)";
 const char* VideoSettings::videoSourceParrotDiscovery   = "Parrot Discovery";
 
+const char* VideoSettings::forceVideoDecoderDefault = "Default";
```

Comment:
`Software Decoder (Default)`

Reply:
The "default" is not necessary software decoder, we just don't change the gstreamer rankings.

Reply:
Ok, that's good. I think we should have an additional option to force software decoder.

Reply:
done